modules/core/src/main/scala/org/scalasteward/core/edit/hooks/HookExecutor.scala

```diff
@@ -72,11 +72,14 @@ object HookExecutor {
     (GroupId("com.codecommit"), ArtifactId("sbt-spiewak-bintray")),
     (GroupId("io.github.nafg.mergify"), ArtifactId("sbt-mergify-github-actions")),
     (GroupId("io.chrisdavenport"), ArtifactId("sbt-davenverse")),
-    (GroupId("org.http4s"), ArtifactId("sbt-http4s-org")),
-    (GroupId("org.typelevel"), ArtifactId("sbt-typelevel")),
     (GroupId("org.typelevel"), ArtifactId("sbt-typelevel-ci-release"))
```

Comment:
`sbt-typelevel-ci-release` is a lower-level plugin which only needs the sbt-github-actions post-update step.